CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Display additional info on the impact of running Firebase extensions with nodejs10 runtime. (#2549)
```

Comment:
Can you write "Node.js 10 runtime" instead?

Reply:
Yep!